cmd/api/pagination/pagination.go

```diff
@@ -28,8 +28,10 @@ const (
 // GetValuesFromContext is a helper function for routes to retrieve the
 // information needed. This is kept here, so it's close to the function
 // that sets these values in the context (Middleware)
-func GetValuesFromContext(context *gin.Context) (string, string, string) {
-	return context.GetString(limitKey), context.GetString(continueIdKey), context.GetString(continueDateKey)
+func GetValuesFromContext(context *gin.Context) (int, string, string) {
+	var limit int
+	limit, _ = strconv.Atoi(context.GetString(limitKey))
```

Comment:
This does make sense as if the limit is present on the context is a valid integer, because we already check on the middleware funciton. However add a comment here explaining why the error is ignored.

Reply:
I just realized that it would make more sense to just do `context.GetInt()` and store it as an integer. PTAL.